app/helpers/ExerciseConfigChecker.php

```diff
@@ -0,0 +1,69 @@
+<?php
```

Comment:
Is there a reason why this file is not in ExerciseConfig subdir?

Reply:
Probably not, I'll move it there.